tests/foreman/cli/test_import.py

```diff
@@ -399,6 +423,9 @@ def tearDown(self):
             u'rm -rf "${HOME}"/.transition_data "${HOME}"/puppet_work_dir'
         )
 
+    def runTest(self):
```

Comment:
OK, googled a little, still don't get it. So `runTest()` is a `unittest` func to actually execute one and the only test in class. So, by overriding it here, you're telling `unittest` to run only `test_import_snippets_default()` from entire `TestImport` class? Isn't it something left from debugging?